input/README.md

```diff
@@ -24,6 +24,8 @@ This folder contains many smaller graphs. They are used in the testing suite and
 | example.graph                    | Format.METIS            | METISGraphReader        | 16                 | 4               | 2               | FALSE    | FALSE    | n/a     |
 | example2.dgs                     | n/a                     | DGSReader               | 161                | n/a             | n/a             | n/a      | n/a      | n/a     |
 | foodweb-baydry.konect            | Format.KONECT           | KONECTGraphReader       | 42847              | 128             | 2137            | TRUE     | TRUE     | n/a     |
+| foodweb-baydry.thrill            | Format.THRILL           | ThrillGraphReader       | 410318              | 128             | 2137            | TRUE     | TRUE     | n/a     |
```

Comment:
The trill file is very large (400 KiB). I don't exactly see why (@michitux ?) Can you take a smaller graph as an example?

Reply:
I have.